packages/prompts/src/index.test.ts

```diff
@@ -2,9 +2,10 @@ import { EventEmitter, Readable, Writable } from 'node:stream';
 import colors from 'picocolors';
 import { afterAll, afterEach, beforeAll, beforeEach, describe, expect, test, vi } from 'vitest';
 import * as prompts from './index.js';
+import progress from './progress-bar.test.js';
 
 // TODO (43081j): move this into a util?
-class MockWritable extends Writable {
+export class MockWritable extends Writable {
```

Comment:
if you catch up from main, you can discard these changes and import from `test-utils.js`